.travis.yml

```diff
@@ -36,6 +36,9 @@ matrix:
   - go: "1.13.x"
     env:
     - HOTROD=true
+  - go: "1.13.x"
```

Comment:
Why do we need a separate matrix step for it? I'd rather include it in the main build.

Reply:
+1 this was just to quickly test the build. I will remove it